vendor/istio.io/api/mcp/v1alpha1/mcp.proto

```diff
@@ -184,3 +183,106 @@ service AggregatedMeshConfigService {
       returns (stream IncrementalMeshConfigResponse) {
   }
 }
+
+// A RequestResource can be sent in two situations:
+//
+// Initial message in an MCP bidirectional change stream
+// as an ACK or NACK response to a previous Resources. In
+// this case the response_nonce is set to the nonce value
+// in the Resources. ACK/NACK is determined by the presence
+// of error_detail.
+//
+// * ACK  (nonce!="",error_details==nil)
+// * NACK (nonce!="",error_details!=nil)
+// * New/Update request (nonce=="",error_details ignored)
+//
+message RequestResources {
+  // The sink node making the request.
+  SinkNode sink_node = 1;
+
+  // Type of resource collection that is being requested, e.g.
+  //
+  // istio/networking/v1alpha3/VirtualService
+  // k8s/<apiVersion>/<kind>
+  string collection = 2;
+
+  // When the RequestResources is the first in a stream, the initial_resource_versions must
+  // be populated. Otherwise, initial_resource_versions must be omitted. The keys are the
+  // resources names of the MCP resources known to the MCP client. The values in the map
+  // are the associated resource level version info.
+  map<string, string> initial_resource_versions = 3;
+
+  // When the RequestResources is an ACK or NACK message in response to a previous RequestResources,
+  // the response_nonce must be the nonce in the RequestResources. Otherwise response_nonce must
+  // be omitted.
+  string response_nonce = 4;
+
+  // This is populated when the previously received resources could not be applied
+  // The *message* field in *error_details* provides the source internal error
+  // related to the failure.
+  google.rpc.Status error_detail = 5;
+}
+
+// Resources do not need to include a full snapshot of the tracked
+// resources. Instead they are a diff to the state of a MCP client.
+// Per resource versions allow sources and sinks to track state at
+// the resource granularity. An MCP incremental session is always
+// in the context of a gRPC bidirectional stream. This allows the
+// MCP source to keep track of the state of MCP sink connected to
```

Comment:
Is this sink = destination?

Reply:
Yes, sink is the destination for configuration updates.